templates/networkpolicies.yaml

```diff
@@ -90,6 +90,19 @@ spec:
       ports:
         - protocol: TCP
           port: 5432
-  # Deny all egress traffic
-  egress: []
+  # Allow egress to istio service mesh. This shouldn't cause any issues if
+  # Istio is not in use.
+  egress:
+  - to:
+    - namespaceSelector:
+        matchLabels:
+          app.kubernetes.io/name: istio-controlplane
+    - podSelector:
```

Comment:
can these be combined? right now, it's checking for the namespace OR pod label, but I think we want an AND here?